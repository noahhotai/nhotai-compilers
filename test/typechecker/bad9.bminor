testfunc: function void ( x: integer, y:integer, z:integer ) = {

          x: integer;
	  y: boolean;

	  z = (x+y)/9;
}