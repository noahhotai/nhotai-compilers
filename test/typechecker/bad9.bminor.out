Program parsed successfully.
z resolves to param 2
x resolves to local 0
y resolves to local 1
Program resolved successfully.
type error: cannot add a integer (x) by an boolean (y).
Program failed to typecheck.
